src/GeometryOps.jl

```diff
@@ -4,16 +4,18 @@ module GeometryOps
 
 using GeoInterface
 using GeometryBasics
-import Tables
 using LinearAlgebra, Statistics
+
+import Tables
 import GeometryBasics.StaticArrays
+import DelaunayTriangulation # for convex hull and triangulation
 import ExactPredicates
 import Base.@kwdef
-
-using GeoInterface.Extents: Extents
+import GeoInterface.Extents: Extents
 
 const GI = GeoInterface
 const GB = GeometryBasics
+const DelTri = DelaunayTriangulation
```

Comment:
```suggestion
const DT = DelaunayTriangulation
```

More consistent with the other package abbreviations

Reply:
Since we won't be using DelaunayTriangulation all that much, I might just drop it in favour of a simple `import` and explicit `DelaunayTriangulation.*`...